Backpack/src/main/res/values/themes.xml

```diff
@@ -9,6 +9,7 @@
     <item name="windowNoTitle">true</item>
     <item name="windowActionBar">false</item>
     <item name="colorButtonNormal">@android:color/white</item>
+    <item name="toolbarStyle">@style/BpkToolbarStyle</item>
```

Comment:
This triggers an update of all docs screenshots

Reply:
This changed the spacing between the title and the nav icon. Have you tested all scrolling animations (including RTL) manually?

Reply:
yep works fine in both